### PR TITLE
SEO-3380 - WPF Control

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Added redirect link for WPF Controls.

Regards,
Sharly A.

### CODE REVIEW

File: wpf/DataGrid/Export-To-PDF.md [hunk 1]
```diff
@@ -289,7 +289,7 @@ document.Save("Sample.pdf");
 {% endhighlight %}
 {% endtabs %}
 
-You can refer [PDF documentation](http://help.syncfusion.com/file-formats/pdf/loading-and-saving-document#saving-a-pdf-document-to-file-system). 
+You can refer [PDF documentation](https://help.syncfusion.com/#saving-a-pdf-document-to-file-system).
```
Comment (gowthamanthavasiyappan) [inline wpf/DataGrid/Export-To-PDF.md:292]: Use this link - https://help.syncfusion.com/file-formats/pdf/open-and-save-pdf-file-in-c-sharp-vb-net#saving-a-pdf-document-to-file-system

File: wpf/DataGrid/Export-To-PDF.md [hunk 2]
```diff
@@ -304,7 +304,7 @@ fileStream.Close();
 {% endhighlight %}
 {% endtabs %}
 
-You can refer [PDF documentation](http://help.syncfusion.com/file-formats/pdf/loading-and-saving-document#saving-a-pdf-document-to-stream). 
+You can refer [PDF documentation](https://help.syncfusion.com/#saving-a-pdf-document-to-stream).
```
Comment (gowthamanthavasiyappan) [inline wpf/DataGrid/Export-To-PDF.md:307]: Use this link - https://help.syncfusion.com/file-formats/pdf/open-and-save-pdf-file-in-c-sharp-vb-net#saving-a-pdf-document-to-stream

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Commented

File: wpf/DataGrid/Export-To-PDF.md (hunk 2 above)
Comment (Sharly-Atieno) [reply wpf/DataGrid/Export-To-PDF.md]: Redirect link added

File: wpf/DataGrid/Export-To-PDF.md (hunk 1 above)
Comment (Sharly-Atieno) [reply wpf/DataGrid/Export-To-PDF.md]: Redirect link added

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**